database/models/clan.js

```diff
@@ -15,8 +15,8 @@ ClanModel.sync();
 
 var Clan = {model: ClanModel};
 
-Clan.findAll = function() {
-  return ClanModel.findAll();
+Clan.findAll = function(query = {}) {
+  return ClanModel.findAll({where: query});
```

Comment:
Hmm interesting point, perhaps find should use findOne to limit results, and findAll can be used this way to have that functionality still exist. I can refactor Users, etc to follow this convention tomorrow.